packages/google-cloud-translate/tsconfig.json

```diff
@@ -2,11 +2,15 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "build",
+    "outDir": "build/cjs",
     "resolveJsonModule": true,
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "sourceMap": false,
+    "module": "CommonJS",
     "lib": [
       "es2018",
-      "dom"
+      "DOM"
```

Comment:
curious what we actually get from this plugin, I would think that DOM would be mostly for web code.

Reply:
We get the definition for window here:
https://github.com/googleapis/google-cloud-node/blob/b9cd90cc4e8eaf732fde676198d8a694e7df200b/packages/google-cloud-translate/src/v3/translation_service_client.ts#L123

Reply:
SGTM!